pkg/repository/prisma/dbsqlc/tenants.sql

```diff
@@ -36,15 +36,16 @@ SELECT
 FROM
     "Tenant" as tenants;
 
+-- name: ControllerPartitionHeartbeat :one
+UPDATE
```

Comment:
did we have problems with this query, i cant remember

Reply:
don't think so